content_delegate.js

```diff
@@ -240,19 +236,28 @@ ContentDelegate.prototype.onDocumentViewSubmit = function (event) {
     let image_string = $('td:contains(Image)').text();
```

Comment:
Can we just make this code:

```js
let document_string = $('th:contains(Description:)').next().text()
```

so that it handles both the DC case
```html
<tr>
<th align="LEFT"><font size="-1" color="DARKBLUE"> Description: </font></th>
<td align="LEFT"><font size="-1" color="DARKBLUE"> Image83-0 </font>
```

and also the BK case:
```html
<tr>
<th align="LEFT"><font size="-1" color="DARKBLUE"> Description: </font></th>
<td align="LEFT"><font size="-1" color="DARKBLUE"> Claim 43976411-0 </font></td>
```

Reply:
Well, when it says Claim, ideally we want that *not* to get parsed anymore. That's a tip off that we're in a place we don't want to be doing uploads.

Reply:
Wellllll... **_ideally_**? It seems…quite a bit removed from _ideally_.

1. Ideally, RECAP should mirror absolutely all of PACER, but we're not there yet. We'll probably never handle deadline dates or case file locations, etc.
2. Practically, RECAP should handle all objects of a particular kind in PACER, not pick and choose. For instance, it should handle all PDFs. As far as I/we know, all PACER PDFs have a DLS and are "Documents" with receipts that belong to a case with a docket number and a `pacer_case_id`. This is part of why I get so incensed about freelawproject/recap#61  and the idea that the server accepts PDFs but withholds them from users pending metadata. That's like…cruel and miserly. So what if there's no metadata? That shouldn't justify my having to pay $3 for a document that someone else has crowdsourced and contributed!
3. However, some of them are Claims, not Images, and belong to a "Claims Register" not a "Docket Report." That requires both server-side and client-side support, which is not on the roadmap.

But it seems to me we are much better off sending them to the server now than holding them back deliberately. It gets us closer to where we want, _ideally_, to be.

Now, maybe something goes wrong if we do that, further down? The obvious is that **(a)** `document_number` and `attachment_number` may not be semantically sane, and **(b)** we are not passing the PDF type (Image or Claim) to `showPdfPage()` and `recap.uploadDocument()`.

I'm not sure (a) is true. So what if we upload `{document_number: "43976411", attachment_number: "0"}` and not `{claim_number: "70-0"}`? Yes, the document number isn't nice and sequential like in DC docket reports, but we already have this in CMECF Appellate and it's fine there (for the IA imported data) so it should be fine here too. And will the `attachment_number` get lost because it's not truthy and we do we care? I don't see why we must care (although perhaps we should).

And I'm not sure (b) is required. Yeah, we should probably adjust the regexp to capture the doctype and pass it to the server, even if the server currently throws it away. And yeah, we should probably fix the server to save it because maybe it'll come in handy later when somebody wants to add support for the Claims Register. 

**(c)** We could also grab the claim number from the Receipt page url (e.g. 
`/cgi-bin/show_doc.pl?caseid=605035&claim_id=43976411&claim_num=70-1&magic_num=MAGIC`) and pass that on. Not required, but probably smart to do, but not on the roadmap and not necessary for PDF crowdsourcing.

But I'm not seeing why we should take steps to make that any harder than it has to be, or to encourage preclusion like this.